<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu Principal</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>xguardian_ico.ico</selectedon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>531</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 39, 122);</string>
   </property>
   <property name="title">
    <string>Painel</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: white;
 border-radius: 3px;</string>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(255, 255, 255);
background-color: rgb(77, 39, 122);</string>
     </property>
     <property name="text">
      <string>Ligar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 900 9pt &quot;Segoe UI&quot;;
background-color: rgb(77, 39, 122);
border-radius: 0px;
</string>
     </property>
     <property name="text">
      <string>BANCO DEV</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>511</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border-radius: 3px;</string>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    background-color: #fff;
    color: #222;
    border: 1px solid #bbb;
    border-radius: 6px;
    padding: 4px 10px;
    font-size: 16px;
}

QComboBox::drop-down {
    background: #fff;
    border-left: 1px solid #bbb;
    width: 25px;
}

QComboBox::down-arrow {
    image: url(:/icons/down_arrow_black.png); /* use um ícone preto, ou remova esta linha */
    width: 14px;
    height: 14px;
}

QComboBox QAbstractItemView {
    background-color: #fff;
    color: #222;
    selection-background-color: #eee;
    selection-color: #111;
    border: none;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #fff;
    color: #222;
    border: 1px solid #bbb;
    border-radius: 6px;
    padding: 6px 16px;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #eee;
    border: 1px solid #999;
}

QPushButton:pressed {
    background-color: #ddd;
}</string>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>511</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 900 12pt &quot;Segoe UI&quot;;
background-color: rgb(77, 39, 122);
border-radius: 0px;</string>
     </property>
     <property name="text">
      <string>Organizações</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>340</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: white;
 border-radius: 3px;</string>
    </property>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(255, 255, 255);
background-color: rgb(77, 39, 122);</string>
     </property>
     <property name="text">
      <string>Ativar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 900 9pt &quot;Segoe UI&quot;;
background-color: rgb(77, 39, 122);
border-radius: 0px;
</string>
     </property>
     <property name="text">
      <string>Robô Testes</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>241</width>
     <height>241</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777</width>
     <height>16777</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(77, 39, 122);
color: rgb(255, 255, 255);
border-radius: 3px;</string>
   </property>
   <property name="text">
    <string>SAIR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
